test/supplemental_genbank_tests/genbank_upload_full_test.py

```diff
@@ -414,6 +484,18 @@ def test_genbanks_to_genomes(self):
             self._load_expected_data("data/genome_curated/assembly_ontology.json"),
         ]
 
+        expected_genome_md5sum = [
+            "b11f26a802d3302dc2648090930bd543",
+            "2ae04b5ede4e27ce1fdd42ff023bf99c",
+            "09b935cb6fc37ea17e36ff4cf72815c1",
+        ]
+
+        expected_assembly_md5sum = [
+            "cd8a65e6af5fcb25bf665e5ed3732607",
+            "cd8a65e6af5fcb25bf665e5ed3732607",
+            "cd8a65e6af5fcb25bf665e5ed3732607",
+        ]
```

Comment:
You manually checked these md5s against the test files before they were uploaded right?

Reply:
yes!!

<img width="918" alt="Screenshot 2024-05-13 at 3 25 30 PM" src="https://github.com/kbaseapps/GenomeFileUtil/assets/22202096/556e9053-6826-49d3-be77-0987e751eee8">